### DISCUSSION TITLE
1.19.2 support

### DISCUSSION THREAD

**Discussion by LunelleVatrinea** [samolego/GolfIV#77] [Q&A] [answered]
Hi there!
I'm looking into setting up a server and felt like an anticheat system would be a good idea, so I stumbled upon this project. Now for my question: does this mod work with 1.19.2 and if so, which mod version should I be using?
Thanks in advance

**Comment by samolego** [comment]
See modrinth / curseforge for minecraft version specific releases.

**Reply by LunelleVatrinea** [reply]
I have already checked both websites, however both of them list a version for 1.19+1.19.1 followed by a release for 1.19.3, skipping 1.19.2 entirely. That's why I was wondering if perhaps a version that is compatible with 1.19.2 wasn't tagged. I probably should've specified that in my original question. Sorry about that

**Reply by samolego** [reply, marked as answer]
Oh, try the 1.19.1 version

**Reply by LunelleVatrinea** [reply]
so far the server has been running without any issues. thanks :)
